packages/core/content-releases/admin/src/pages/SettingsPage.tsx

```diff
@@ -15,31 +82,112 @@ export const SettingsPage = () => {
           }
         )}
       </Page.Title>
-      <Page.Main>
-        <Form method="PUT">
-          <Layouts.Header
-            primaryAction={
-              <Button disabled={false} loading={false} startIcon={<Check />} type="submit" size="L">
-                {formatMessage({
-                  id: 'global.save',
-                  defaultMessage: 'Save',
-                })}
-              </Button>
-            }
-            title={formatMessage({
-              id: 'content-releases.pages.Settings.releases.title',
-              defaultMessage: 'Releases',
-            })}
-            subtitle={formatMessage({
-              id: 'content-releases.pages.Settings.releases.description',
-              defaultMessage: 'Create and manage content updates',
-            })}
-          />
-          <Layouts.Content>
-            <div>TODO: define the Settings page content (not part of this US)</div>
-          </Layouts.Content>
+      <Page.Main aria-busy={isLoadingSettings} tabIndex={-1}>
+        <Form
+          method="PUT"
+          initialValues={
+            data?.data.defaultTimezone
+              ? data.data
+              : {
+                  defaultTimezone: null,
+                }
+          }
+          onSubmit={handleSubmit}
+          validationSchema={SETTINGS_SCHEMA}
+        >
+          {({ modified, isSubmitting }: { modified: boolean; isSubmitting: boolean }) => {
+            return (
+              <>
+                <Layouts.Header
+                  primaryAction={
+                    <Button
+                      disabled={!modified || isSubmittingForm}
+                      loading={isSubmitting}
+                      startIcon={<Check />}
+                      type="submit"
+                      size="L"
```

Comment:
I don't think size L is needed, the button doesn't look this big to me in the designs

Reply:
ok